Answers.md

```diff
@@ -2,10 +2,22 @@
 
 1. What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
-1. What does it mean to think in react?
+    It is a JS library for building user interfaces, It helps by seperating our reusuable components into their own component files.

```

Comment:
I think the best part of react is that it scales well for large applications using the virtual DOM.  Being able to separate functions into components would be second, and being able to write JS and HTML in the same file would be 3rd.

Virtual DOM is a key word and recruiters/managers WILL ask you about it.